ROADMAP.md

```diff
@@ -0,0 +1,35 @@
+# Power Platform Terraform Provider Roadmap
+
+This roadmap outlines the planned direction for the Power Platform Terraform Provider over the next 12 months. It serves as a high-level guide for contributors and users, providing transparency into where the project is headed while maintaining flexibility to adapt to community needs and evolving requirements.
+
+## Key Themes and Planned Direction
+
+| Key Theme | Planned Direction (12-month horizon) |
+|-----------|--------------------------------------|
+| **Quality & Reliability** | Increase test coverage across all resources and data sources, reduce flaky acceptance tests through improved test isolation and mock strategies, and formalize a predictable release cadence with automated quality gates. Focus on stability improvements for existing resources before adding new functionality. |
+| **Security Hardening** | Strengthen binary-signing and SBOM (Software Bill of Materials) practices in our release pipeline, integrate stricter static and dynamic analysis tools into CI/CD workflows, and address any newly reported CVEs promptly. Enhance authentication mechanisms and improve security documentation for users. |
+| **AI & Copilot Studio Support** | Expand resources and data sources that simplify building AI-powered solutions on Power Platform, including enhanced support for Copilot Studio configurations, AI Builder components, and related Power Platform AI services. Provide comprehensive examples and documentation for AI use cases. |
+| **Community-Driven Improvements** | Prioritize fixes for issues raised by users in the GitHub repository, improve error messages and diagnostics to enhance the user experience, and host periodic community discussions to gather feedback on feature priorities. Streamline the contribution process and expand documentation for new contributors. |
+
```

Comment:
add another item about Governance and Environment management features.  We would consider extending or adding new resources and data sources with new features to support environment governance, policies, or large-scale environment management.  one example would be disaster recovery settings on environments.